webpack.config.babel.js

```diff
@@ -35,7 +35,10 @@ const basePlugins = [
   new webpack.NoErrorsPlugin(),
   new webpack.optimize.DedupePlugin(),
   new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify(ENV)
+    'process.env': {
+      'NODE_ENV': JSON.stringify(ENV),
+      'ONFIDO_URL': JSON.stringify(process.env.ONFIDO_URL || 'https://apidev.onfido.com')
```

Comment:
This is not consistent with how we set the SDK URL.

Should it default to the staging environment?